mlperf_common/mountcheck.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import os
+import json
+import subprocess
+
+def du(path):
+    return subprocess.check_output(['du','-sk', path]).split()[0].decode('utf-8')
+
+def getMountInfo(paths_to_verify):
+
+    paths_to_verify = paths_to_verify.split(",")
+
+    for dir_path in paths_to_verify:
+        if not os.path.exists(dir_path):
+            print("Error: path $dir_path is incorrect")
+            sys.exit(1)
+
+    cont_mount_info = []
+    for dir_path in paths_to_verify:
+        records = []
+        for root, dirs, files in os.walk(dir_path):
+            files = [f for f in files if not f[0] == '.']
+            dirs[:] = [d for d in dirs if not d[0] == '.']
+            root_suffix = root[len(dir_path)+1:] # root without dir_path prefix
+            records.append(
+                { 
+                    "path": root_suffix, 
+                    "elements": len(files) + len(dirs),
+                    "dir_size": int(du(root))
+                }
+            )
+        
+        records.sort(key=lambda x: x['path'], reverse=True)
+        cont_mount_info.append(
+            {
+                "name": dir_path,
+                "subdirs": records,
+            }
+        )
+        
+    cont_mount_info_json = json.dumps(cont_mount_info, indent=4)
```

Comment:
return the `cont_mount_info` data structure rather than printing a json encoded string

Reply:
I'm doing both